### ISSUE TITLE
if then else with empty branches gets ignored

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/asdec#46]
```
What steps will reproduce the problem?
1. Adding if then else and if then else inside.

What is the expected output? 
if (_loc1_ == "test1")
{
   trace("test 1");
   if (_loc2_ == "test2")
   {
     trace("test 2");
   }
   else {
     trace("else 2");
   }
}
else {

trace("else 1");
} 

What do you see instead?
if(_loc2_=="test1")
{
}

What version of the product are you using? On what operating system?
1.4.1 on windows 7 64 bit

Could you provide the SWF file you have problem with?
If the answer is yes, then please attach it here or send me it via email.
Could you at least attach PCode source?

Do you have the original source code which produced the wrong
decompilation? If yes, then please attach it.

Please provide any additional information below.


```

Original issue reported on code.google.com by `capas...@gmail.com` on 21 Mar 2013 at 5:13

Attachments:
- [ififthenelse.zip](https://storage.googleapis.com/google-code-attachments/asdec/issue-46/comment-0/ififthenelse.zip)

**Comment by GoogleCodeExporter** [comment]
```
hi,
the trace commands are ignored during compilation with your compiler.
The decompiler ignores if commands when there are no commands in both if and 
else branches, e. g.  
pushbyte 1
pushbyte 2
ifeq loc1
loc1:

However in some cases the if should be visible, because the condition can have 
side-effect, like
if(exec()){

}else{

}

I will look on it, and fix it.

The code should then look like this:

if(_loc2_=="test1")
{
   if (_loc2_ == "test2")
   {
   }
}
```

Original comment by `jindra.p...@gmail.com` on 21 Mar 2013 at 6:36
- Changed title: **if then else with empty branches gets ignored**
- Changed state: **Accepted**
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Hi,
this is fixed in version 1.4.2, try it.
```

Original comment by `jindra.p...@gmail.com` on 24 Mar 2013 at 11:53
- Changed state: **Fixed**
